test/test_general.py

```diff
@@ -1,82 +1,116 @@
 '''
 A general test script that ensures PaCMAP can be successfully loaded.
 '''
-import sklearn
-import pacmap
+from sklearn.datasets import fetch_openml
+from pacmap import pacmap
```

Comment:
We're no longer copying the `pacmap.py` file from `source/pacmap/pacmap.py` and instead testing directly on `pacmap` module, so we need to import slightly differently in tests to get access to `.save()` and `.load()`

On the upside, the tests now look more like how a regular user would use the library, so it's better documentation :)

Reply:
Thanks for the note here, the updated import is fine and clear to use.